cli/cluster/cluster.go

```diff
@@ -95,36 +98,39 @@ func collectTarantoolConfig(collectors libcluster.CollectorFactory,
 
 		if endpoint.Params.Transport == "" || endpoint.Params.Transport != "ssl" {
 			opts = append(opts, tarantoolOpts{
+				addr: fmt.Sprintf("%s://%s", network, address),
+				dialer: tarantool.NetDialer{
+					Address:  fmt.Sprintf("%s://%s", network, address),
+					User:     endpoint.Login,
+					Password: endpoint.Password,
+				},
 				opts: tarantool.Opts{
-					User:       endpoint.Login,
-					Pass:       endpoint.Password,
-					Transport:  endpoint.Params.Transport,
 					SkipSchema: true,
 				},
-				addr: fmt.Sprintf("%s://%s", network, address)})
+			})
 		} else {
 			opts = append(opts, tarantoolOpts{
+				addr: fmt.Sprintf("%s://%s", network, address),
+				dialer: tlsdialer.OpenSSLDialer{
+					Address:     fmt.Sprintf("%s://%s", network, address),
```

Comment:
Probably we could calculate address only once in the upper level of the condition.